apps/submit-app.md

```diff
@@ -6,26 +6,121 @@ title: Submit an app
 
-Let's go submitting your app.
+## Let's go submit your app
 
-1. Within your test shop, click the tab **Developer Apps**.
+1. Within the administration area of your test shop, click the tab **Apps & Themes**.
+In the drop down menu, choose the field **Developer apps** and click on your app in the list below.
+    {% image click-apps.png %}{% endimage %}
 2. Click the button **Submit app**.
-3. Fill out the submission form.
+    {% image submit-app.png %}{% endimage %}
+3. You will then be asked to fill out the submission form.
+    {% image submission-form.png %}{% endimage %}
 
-This is what we need to thoroughly test the functionality of your app.
+## Walk through the submission form
 
-| What?              | Why?                                                                                                    |
-|--------------------|---------------------------------------------------------------------------------------------------------|
-| General information      | We need your contact details for proper identification.                                           |
-| Details about your app   | Facts like app settings, language, price of your service, support information etc are relevant for the submission process.|
-| Marketing information    | Your app goes live and needs to be placed nicely in the ePages App Store. |
+The submission form validates for the **Client ID** and **Client Secret**.
+However, we'll ask you to provide us with additional information and graphics to get accepted into the App Store.
+Asterisks ( * ) indicate a required field.
 
-Before submitting your app, review all the information that you provide to us and also make sure that the **Application Callback URL** and **Application Notification URL** are correct.
+### About you
+
+We need your contact details for proper identification and to get in touch with you.
+
+### About your app
+
+**Client ID** and **Client Secret** will be pre-filled, all other app details have to be provided by you.
+
+#### Authorisation
+
+Tick the respective check box(es) to set up the scope for your app.
+This indicates the type of calls your app will make on the REST API.
+
+#### App details
+
+English app details are mandatory.
+If a merchant switches his shop to a language your app does not support, the fallback language is English.
+If you decide to offer your app in multiple languages, tick the respective check boxes.
+You will then also have to fill out all mandatory fields for the additional language(s).
+
+Requirements: describe the requirements for using your service, e.g. pricing.
+
+{% highlight text %}
+Example: Subscription model: $19.99 per month.
+{% endhighlight %}
+
+Developer or company name: provide the name of the individual(s) or company which developed the app.
+
+Features: provide three key features of your app, which will be shown as an list.
```

Comment:
"which will be shown as a list" - not "AN" if i recall my english grammar correctly